cmd/bendo/main.go

```diff
@@ -7,31 +7,51 @@ import (
 	"os/signal"
 	"syscall"
 
+	"github.com/BurntSushi/toml"
 	"github.com/ndlib/bendo/items"
 	"github.com/ndlib/bendo/server"
 	"github.com/ndlib/bendo/store"
 )
 
+//Config info needed for Bendo
+
+type bendoConfig struct {
+	StoreDir   string
+	Tokenfile  string
+	CacheDir   string
+	CacheSize  int64
+	PortNumber string
+	PProfPort  string
+	Mysql      string
+	Cow        string
+}
+
 func main() {
-	var (
-		storeDir   = flag.String("storage", ".", "location of the storage directory")
-		tokenfile  = flag.String("user-tokens", "", "file containing allowable user tokens")
-		cacheDir   = flag.String("cache-dir", "", "directory to store the blob cache")
-		cacheSize  = flag.Int64("cache-size", 100, "the maximum size of the cache in MB")
-		portNumber = flag.String("port", "14000", "Port Number to Use")
-		pProfPort  = flag.String("pfport", "14001", "PPROF Port Number to Use")
-		mysql      = flag.String("mysql", "", "Connection information to use a MySQL database")
-		cow        = flag.String("copy-on-write", "", "External bendo server to mirror content from")
-	)
+
+	// Start with the Default values
+	//config := bendoConfig{".", "", "", 100, "14000", "14001", "", ""}
+	var config bendoConfig
```

Comment:
Setup the default values. You can use the

    config := bendoConfig{
        PortNumber: "14000",
    }

syntax to make it easier to read.